NitroxClient/Communication/Client.cs

```diff
@@ -66,5 +93,25 @@ public void PacketSentSuccessful(IAsyncResult ar)
         {
 
         }
+
+        public bool isConnected()
+        {
+            if (connection == null)
+            {
+                return false;
+            }
+            return connection.Open;
+        }
+
+        private void outputMessage(String msg)
+        {
+            if (testClient)
```

Comment:
Maybe for the future we can pass in a logger as this looks be the only thing testClient is used for